Volume/Views/Orgs Admin/OrgsAdminView.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  OrgsAdminView.swift
+//  Volume
+//
+//  Created by Vin Bui on 11/3/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct OrgsAdminView: View {
+
+    // MARK: - Properties
+
+    @Environment(\.dismiss) var dismiss
+    @EnvironmentObject private var networkState: NetworkState
+    @StateObject private var viewModel = ViewModel()
+
+    let organization: Organization?
+
+    // MARK: - Constants
+
+    private struct Constants {
+        static let pastTabWidth: CGFloat = 64
+        static let sectionSpacing: CGFloat = 24
+        static let sidePadding: CGFloat = 16
+        static let topPadding: CGFloat = 28
+        static let upcomingTabWidth: CGFloat = 96
+    }
+
+    // MARK: - UI
+
+    var body: some View {
+        ScrollView(.vertical, showsIndicators: false) {
+            VStack(alignment: .leading, spacing: Constants.sectionSpacing) {
+                titleUploadSection
+                slidingTabBar
+                flyersSection
+
+                Spacer()
```

Comment:
Not sure if this is convention but I thought you put line breaks before and after ` Spacer()` 's?

Reply:
There isn't really a convention for it. I usually put linebreaks depending on what that spacer is doing. Since this one is pushing everything up and there's nothing below it, I don't add a line break below since it's the last thing.